random_string.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
```

Comment:
Please implement OOPS here.

Additionally let's try to create few class files and keep all our methods there based on operation and requirement instead of creating separate files for each method